tests/gadget-zero/test_gadget0.py

```diff
@@ -3,8 +3,11 @@
 import usb.core
 import usb.util as uu
 import logging
+import struct
+import time
 
 import unittest
+import pytest
```

Comment:
What are trying here? mixing different styles of tests in the same file is pretty confusing to say the least.